### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by arcturus2**
### Description of the Change

* update to missing what's new  entries
* update to missing CHANGELOG entries
* A README update to change the instructions to use NB 12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,10 +75,10 @@ To build Constellation from source code do the following:
 [Linux 64 bit](https://cdn.azul.com/zulu/bin/zulu11.39.15-ca-fx-jre11.0.7-linux_x64.tar.gz)
 or [MacOSX 64 bit](https://cdn.azul.com/zulu/bin/zulu11.39.15-ca-fx-jre11.0.7-macosx_x64.tar.gz)
 
-* Download [NetBeans 11.3](https://netbeans.apache.org/download/nb113/nb113.html)
-* Update `netbeans_jdkhome` in netbeans.conf to point to the Azul Zulu JDK you downloaded (e.g. `C:\Program Files\NetBeans-11.3\netbeans\etc`)
+* Download [NetBeans 12](https://netbeans.apache.org/download/nb120/nb120.html)
+* Update `netbeans_jdkhome` in netbeans.conf to point to the Azul Zulu JDK you downloaded (e.g. `C:\Program Files\NetBeans-12\netbeans\etc`)
 * NetBeans 11 have dropped support for the Java Help system but is still required by Constellation. Until we find a solution to #15 there are 2 jar files that have to be manually copied into the NetBeans installation folder.
```
Comment (aldebaran30701) [inline README.md:80]: Netbeans 12 * has dropped support

File: README.md [hunk 2]
```diff
@@ -75,10 +75,10 @@ To build Constellation from source code do the following:
...
-  * Download [org-netbeans-modules-javahelp.jar](https://github.com/constellation-app/third-party-dependencies/blob/master/NetBeans%20Help/org-netbeans-modules-javahelp.jar?raw=true) to `C:\Program Files\NetBeans 11.3\platform\modules` (using Windows as an example). Note that you will need to override this file when prompted.
+  * Download [org-netbeans-modules-javahelp.jar](https://github.com/constellation-app/third-party-dependencies/blob/master/NetBeans%20Help/org-netbeans-modules-javahelp.jar?raw=true) to `C:\Program Files\NetBeans 12\platform\modules` (using Windows as an example). Note that you will need to override this file when prompted.
   * Download [jhall-2.0_05.jar](https://github.com/constellation-app/third-party-dependencies/blob/master/NetBeans%20Help/jhall-2.0_05.jar?raw=true) to `C:\Program Files\NetBeans 11.3\netbeans\platform\modules\ext` (using Windows as an example)
```
Comment (aldebaran30701) [inline README.md:82]: Files\NetBeans 11.3 - > 12

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt [hunk 3]
```diff
@@ -1,15 +1,59 @@
...
+<p>A bug preventing running the <strong>Select Top N</strong> from the Data Access View has been fixed.</p>
+
+== 2020-05-21 Schema View
+<p>The Node and Transaction type tabs in the Schema View are now searchable.</p>
+
+== 2020-05-21 Hierarchical Clustering Improvements
+<p>A bug preventing the running of the hierachical clustering sometimes has been fixed.</p>
+
+== 2020-05-21 Merge Nodes by Geospatial Distance
+<p>A bug preventing running the <strong>Merge Nodes</strong> &gt; <strong>Geospatial Distance</strong> from the Data Access View has been fixed.</p>
```
Comment (aldebaran30701) [inline CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt:30]: <p>A bug preventing running the Merge Nodes and Geospatial Distance <strong>plugins</strong> from the Data Access View has been fixed.</p>

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt [hunk 4]
```diff
@@ -1,15 +1,59 @@
...
+<p>A bug preventing running the Clustering %gt; Info Map plugins has been fixed.</p>
+
+== 2020-06-09 Select Top N Bug Fixed
+<p>A bug preventing running the <strong>Select Top N</strong> from the Data Access View has been fixed.</p>
+
+== 2020-05-21 Schema View
+<p>The Node and Transaction type tabs in the Schema View are now searchable.</p>
+
+== 2020-05-21 Hierarchical Clustering Improvements
+<p>A bug preventing the running of the hierachical clustering sometimes has been fixed.</p>
```
Comment (aldebaran30701) [inline CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt:27]: running of the hierachical clustering <strong>plugin</strong> sometimes

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt [hunk 5]
```diff
@@ -1,15 +1,59 @@
...
+<ul>
+<li>The filtering options from the preview table have been fixed so that doing <strong>Row !=1</strong> works again.</li>
+<li>A UX improvement has been made to allow you to import files of one type per session. After importing the first file, the <strong>Import File Parser</strong> is grayed out</li>
+</ul>
+
+== 2020-06-18 Info Map Bug Fixed
+<p>A bug preventing running the Clustering %gt; Info Map plugins has been fixed.</p>
+
+== 2020-06-09 Select Top N Bug Fixed
+<p>A bug preventing running the <strong>Select Top N</strong> from the Data Access View has been fixed.</p>
```
Comment (aldebaran30701) [inline CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt:21]: A bug preventing running the Select Top N <strong>plugin</strong>  from the Data Access View has been fixed.

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt [hunk 6]
```diff
@@ -1,15 +1,59 @@
...
+
 == 2020-05-11 Two New Arrangement Options
 <p>Two new arrangement options are available via the Arrange menu and toolbar. These options are horizontal and vertical lines. Selecting these options allow selected (or all) elements to be arranged in either horizontal or vertical lines.</p>
 
-== 2020-05-11 Delimiter importer
+== 2020-05-11 Delimiter Importer
 <p>Added feedback dialogs for the Delimiter importer, showing information about a successful import and any failures.</p>
 
+== 2020-05-11 Timeline View
+<p>A bug resulting in the summary box shrinking when dragged passed the summary bounds has been fixed.</p>
```
Comment (antares1470) [inline CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt:52]: dragged past*

File: README.md (hunk 1 above)
Comment (arcturus2) [reply README.md]: Thanks, updated.

File: README.md (hunk 2 above)
Comment (arcturus2) [reply README.md]: Thanks, updated.

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt (hunk 3 above)
Comment (arcturus2) [reply CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt]: Thanks, updated.

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt (hunk 4 above)
Comment (arcturus2) [reply CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt]: Thanks, updated.

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt (hunk 5 above)
Comment (arcturus2) [reply CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt]: Thanks, updated.

File: CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt (hunk 6 above)
Comment (arcturus2) [reply CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt]: Thanks, updated.

**Pull request merged**